easybuild/easyconfigs/m/MbedTLS/MbedTLS-2.26.0-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,45 @@
+# Author: Jasper Grimm
+
+easyblock = 'ConfigureMake'
+
+name = 'MbedTLS'
+version = '2.26.0'
+
+homepage = 'tls.mbed.org'
```

Comment:
@jfgrimm there is a non-written rule for the `homepage` to have a full URL with protocol. Since it is a very minor change, I pushed the fix myself. I hope that you don't mind.